src/write.rs

```diff
@@ -431,14 +431,30 @@ where
     /// Remove the given value from the value-bag of the given key.
     ///
     /// The updated value-bag will only be visible to readers after the next call to `refresh()`.
+    #[deprecated(since = "11.0.0", note = "Renamed to remove_value")]
```

Comment:
Is this the right version? I imagine the alpha you're on is therefore pre-`11.0.0`? In which case ... do we need to backwards compatible? I imagine it's a major release so ⚖️. Unless the current `11.0.0-alpha` is meant to be between `11.0.0` and `11.1.0` or some such?

Reply:
You're totally right that we don't _have_ to, since we're about to make a breaking release anyway. However, I think having one major version with the methods deprecated is worthwhile. Makes transitioning a little nicer :)